doc/content/source/materials/ConcreteThermalMoisture.md

```diff
@@ -376,73 +396,101 @@ mechanisms separately. Instead, they tend to reproduce the general combined tren
 
 Calculation of $D_h$ starts with the calculation of a reference moisture diffusivity, $D_{h,0}$, at a given temperature, $T$, and relative humidity, $H$. Three reference moisture diffusivity $D_{h,0}$ models are implemented as:
 
-1. [!cite](mensi1988sechage)
+1. [!cite](xi_moisture_1994_a)
 
-!equation id=Dh0_mensi
-D_{h,0} = Ae^{BC}
+  [!cite](xi_moisture_1994_a) evaluate the moisture diffusivity for concrete according to
 
-where
+!equation id=Dh
+D_{h,0} = D_{Hcp}\left[1 + \frac{g_i}{\frac{1-g_i}{3} + \frac{1}{\frac{D_{Hagg}}{D_{Hcp}}-1}}\right],
 
-$D_{h,0}$ = humidity diffusion coefficient of concrete\\
-$A = 3.8 \times 10^{-13} m^2/s$ \\
-$B = 0.05$ \\
-$C$ = free water content in L/m$^3$
+  where
 
-$C$ is a function of relative humidity, $H$, in concrete as given by
+  $D_{h}$ = humidity diffusion coefficient of concrete \\
+  $D_{Hcp}$ = humidity diffusion coefficient of cement paste \\
+  $D_{Hagg}$ = humidity diffusion coefficient of aggregate (= 0 m$^2$/s) \\
```

Comment:
I'd remove the '=0', since you say that below, and it is kind of confusing to mix that with the units in the parentheses.

Reply:
Can you still address this?